e2e/helpers/api_helper.js

```diff
@@ -88,6 +88,17 @@ const createCase = async (user, caseName) => {
   return await response.json();
 };
 
+const getUser = async (user) => {
+  const authToken = await getAuthToken(config.systemUpdateUser);
```

Comment:
`config.systemUpdateUser` is the default value so you don't have to pass it

Reply:
done